iaso/api/org_unit_change_requests.py

```diff
@@ -40,14 +45,15 @@ def get_current_org_unit_groups(self, obj: OrgUnitChangeRequest):
         return [group.name for group in obj.org_unit.groups.all()]
```

Comment:
Are we really only outputting the name of the group? This could totally change over time.

Reply:
That's [what we agreed on](https://github.com/BLSQ/iaso/blob/1daf727c40a4546c78c0070270f1a57f03df0fb1/docs/pages/dev/reference/API/org_unit_registry.md?plain=1#L116) for this read-only serializer.

It can be changed easily in case of problems.